scriptworker/cot/verify.py

```diff
@@ -242,7 +242,11 @@ def task(self, task):
         self.decision_task_id = get_decision_task_id(self.task)
         self.parent_task_id = get_parent_task_id(self.task)
         self.worker_impl = guess_worker_impl(self)
-        self.is_try = is_try(self.task, source_env_prefix=self.context.config['source_env_prefix'])
+
+    @property
+    def is_try_or_pull_request(self):
+        """bool: the task is either a try or a pull request one."""
+        return is_try_or_pull_request(self.context, self.task)
```

Comment:
We may end up calling this multiple times, but most likely that won't be too expensive.

Reply:
Yeah, I was worried about that too. I don't think it is either